index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="website" content="NF website Hugh" />
+    <title>My First NF Website</title>
+
+    <!--   <link href="./style.css" rel="stylesheet" /> -->
+  </head>
+
+  <body>
+    <header>
+      <h1>Hugh Burgess</h1>
+      <nav>
+        <ul>
+          <a href="#Home"><li>Home</li></a>
+          <a href="#Contact"><li>Contact</li></a>
+          <a href="#Career"><li>Career</li></a>
+        </ul>
+      </nav>
+
+      <br />
+
+      <img src="/images/profile-picture.png" alt="Picture of Hugh" />
+    </header>
+    <section>
+      <h2 id="Career">About Me</h2>
+      <article>
+        <p>
+          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quo, saepe
+          aperiam alias vitae quod autem iusto untur itaque in vel quidem dicta
+          temporibus cupiditate voluptatum numquam velit perferendis maiores
+          animi!
+        </p>
+      </article>
+    </section>
+
+    <section>
+      <div class="box">
+        <p>Mobile: 07234823492342213434</p>
+        <p>Email: <a href="">hello@me.com</a></p>
+      </div>
+    </section>
+
+    <article style="text-align: center" class="links" id="Contact">
```

Comment:
I "like" that you have styles in a separate file, so I think you should have a class for `text-align: center
` as well

Also I think you're missing a headline, see: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article#usage_notes